### ISSUE TITLE
Hyperlinks are not rendered as tag a

### ISSUE THREAD

**Issue by takenokos** [zVolodymyr/docxjs#34]
Hyperlinks are not rendered as tag `a`

***
 This is part of the `document.xml`

```xml
    <w:p>
      <w:pPr>
        <w:ind w:firstLine="420" w:firstLineChars="0"/>
      </w:pPr>
      <w:r>
        <w:rPr>
          <w:color w:val="auto"/>
          <w:u w:val="none"/>
        </w:rPr>
        <w:fldChar w:fldCharType="begin"/>
      </w:r>
      <w:r>
        <w:rPr>
          <w:color w:val="auto"/>
          <w:u w:val="none"/>
        </w:rPr>
        <w:instrText xml:space="preserve"> HYPERLINK \l &quot;_# test title&quot; </w:instrText>
      </w:r>
      <w:r>
        <w:rPr>
          <w:color w:val="auto"/>
          <w:u w:val="none"/>
        </w:rPr>
        <w:fldChar w:fldCharType="separate"/>
      </w:r>
      <w:r>
        <w:rPr>
          <w:rStyle w:val="6"/>
        </w:rPr>
        <w:t>test</w:t>
      </w:r>
      <w:r>
        <w:rPr>
          <w:color w:val="auto"/>
          <w:u w:val="none"/>
        </w:rPr>
        <w:fldChar w:fldCharType="end"/>
      </w:r>
    </w:p>
```

```xml
    <w:p>
      <w:pPr>
        <w:ind w:left="420" w:leftChars="0" w:firstLine="420"/>
        <w:rPr>
          <w:rStyle w:val="6"/>
          <w:i/>
          <w:iCs/>
          <w:lang w:eastAsia="zh-Hans"/>
        </w:rPr>
      </w:pPr>
      <w:r>
        <w:rPr>
          <w:rFonts w:hint="eastAsia"/>
          <w:i/>
          <w:iCs/>
          <w:color w:val="auto"/>
          <w:u w:val="single"/>
          <w:lang w:eastAsia="zh-Hans"/>
        </w:rPr>
        <w:fldChar w:fldCharType="begin"/>
      </w:r>
      <w:r>
        <w:rPr>
          <w:rFonts w:hint="eastAsia"/>
          <w:i/>
          <w:iCs/>
          <w:color w:val="auto"/>
          <w:u w:val="single"/>
          <w:lang w:eastAsia="zh-Hans"/>
        </w:rPr>
        <w:instrText xml:space="preserve"> HYPERLINK &quot;https://www.baidu.com/&quot; </w:instrText>
      </w:r>
      <w:r>
        <w:rPr>
          <w:rFonts w:hint="eastAsia"/>
          <w:i/>
          <w:iCs/>
          <w:color w:val="auto"/>
          <w:u w:val="single"/>
          <w:lang w:eastAsia="zh-Hans"/>
        </w:rPr>
        <w:fldChar w:fldCharType="separate"/>
      </w:r>
      <w:r>
        <w:rPr>
          <w:rStyle w:val="6"/>
          <w:rFonts w:hint="eastAsia"/>
          <w:i/>
          <w:iCs/>
          <w:lang w:eastAsia="zh-Hans"/>
        </w:rPr>
        <w:t>敬请期待</w:t>
      </w:r>
    </w:p>
```
[案例1.docx](https://github.com/zVolodymyr/docxjs/files/8549212/1.docx)

**Comment by zVolodymyr** [comment]
At this point, library doesn't support rendering of field instructions, and render only field result that is generated by editing application (it goes after `<w:fldChar w:fldCharType="separate"/>` field, more details about complex - http://officeopenxml.com/WPfields.php). Weird thing is that your document doesn't contain `<w:hyperlink>` element in result for hyperlink fields, and it causes missed `<a/>` tags in render result.

**Comment by takenokos** [comment]
> At this point, library doesn't support rendering of field instructions, and render only field result that is generated by document processing software like Word, etc (it goes after `<w:fldChar w:fldCharType="separate"/>` field, more details about complex - http://officeopenxml.com/WPfields.php). Weird thing is that your document doesn't contain `<w:hyperlink>` element in result for hyperlink fields, and it causes missed `<a/>` tags in render result.

Probably because I created the file in `wps`.
When I create an `http` hyperlink with `msoffice`, the xml does have `<w:hyperlink >`, and the `a` tag is rendered correctly, but it still doesn't render `href` attribute.
I may need to adapt it to my needs.
thanks.
[便携频谱仪.docx](https://github.com/zVolodymyr/docxjs/files/8559198/default.docx)

**Comment by zVolodymyr** [comment]
Last issue just have been fixed and be released with next release

**Comment by zVolodymyr** [comment]
Partially fixed in version 0.1.11
